src/clojure_lsp/handlers.clj

```diff
@@ -41,7 +41,9 @@
        (if (> (quot (- (System/nanoTime) ~'_time) 1000000) 60000) ; one minute timeout
          (log/warn "Timeout waiting for changes for body")
          (if (:processing-changes @db/db)
-           (recur)
+           (do
+             (Thread/sleep 100)
```

Comment:
I wonder how much this will affect all features that use this macro, should this be something smaller? not sure what value, but 100ms sounds a little too much for some features?

Reply:
That's an ad hoc number indeed. What's your suggestion? Maybe it's better to have an incremental backoff?

Reply:
yeah, something exponential sounds better, just not sure what buckets should we use

Reply:
we could leave the 200 as max bucket and start with little buckets of 50? not sure it worths though

Reply:
It looks like we didn't have problems when there's no backoff, so I think we can simply start from 2^0, 2^1, ... and set 200ms as max.

Reply:
sounds like a good start